storage_controller/src/service/chaos_injector.rs

```diff
@@ -47,6 +48,16 @@ impl ChaosInjector {
                 .get_mut(victim)
                 .expect("Held lock between choosing ID and this get");
 
+            if !matches!(shard.get_scheduling_policy(), ShardSchedulingPolicy::Active) {
```

Comment:
For completeness, I would allow `Essential` too. It would mimic prod a bit better.

Reply:
My intent here is to limit the chaos task to "normal" shards only, and interpret all other states as "a human is up to something, I won't touch it".